src/main/java/seedu/address/model/Model.java

```diff
@@ -15,6 +15,9 @@
     /** {@code Predicate} that always evaluate to true */
     Predicate<FlashCard> PREDICATE_SHOW_ALL_FLASHCARDS = unused -> true;
 
+    /** {@code Predicate} that always evaluate to true */
+    Predicate<Deadline> PREDICATE_SHOW_ALL_DEADLINES = unused -> true;
```

Comment:
Styling - is it all still all caps for non final static? Might wanna check

Reply:
This was originally from the AB3, so I think is okay.